eng/pipelines/docindex.yml

```diff
@@ -0,0 +1,51 @@
+resources:
+  repositories:
+    - repository: azure-sdk-tools
+      type: github
+      name: azure/azure-sdk-tools
+      endpoint: azure
+
+jobs:
+  - job: GenerateDocIndex
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: windows-2019
+    steps:
+      - task: UsePythonVersion@0
+        displayName: 'Use Python 3.6'
+        inputs:
+          versionSpec: '3.6'
+      - template: eng/pipelines/templates/scripts/replace-relative-links.yml@azure-sdk-tools
+        parameters:
+          TargetFolder: '.'
+          RootFolder: '.'
+          BuildSHA: $(Build.SourceVersion)
+          RepoId: 'Azure/azure-sdk-for-net'
+      - pwsh: |
+          Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://github.com/dotnet/docfx/releases/download/v2.43.2/docfx.zip" `
+          -OutFile "docfx.zip" | Wait-Process; Expand-Archive -Path "docfx.zip" -DestinationPath "./docfx/"
```

Comment:
Create a tools directory variable in this pipeline and use it here and pass it into your script that uses the tool.

Reply:
Am using the $(Build.BinariesDirectory) which i already pass into the script